### PR TITLE
Adds Developer Documentation

### PR DESCRIPTION
**Pull request by thatarchguy**
Most likely there are spelling errors :sweat_smile: 

Signed-off-by: Kevin <kevin@stealsyour.pw>

### CODE REVIEW

**Comment by thatarchguy** [comment]
Closes #123

**Comment by PrahlM93** [comment]
Looks good. Can we mention mkdocs and Vagrant in this?

Also, we should probably have Apache automatically reload the source code when it is changed in our Vagrant setup (https://code.google.com/archive/p/modwsgi/wikis/ReloadingSourceCode.wiki)

**Comment by thatarchguy** [comment]
> Can we mention mkdocs and Vagrant in this?

Where specifically?

>Also, we should probably have Apache automatically reload the source code

sgtm, make it a new issue

**Comment by PrahlM93** [comment]
Sorry, I didn't see the other files. I'm going to take a look and format them to look good with mkdocs.

**Comment by PrahlM93** [comment]
@thatarchguy, I reformatted and removed the Ansible without Vagrant section because it'll just be easier if we only support one method to spin up a dev environment. If you're good with the changes, merge it and we can update the docs.

File: docs/Development/DevEnvironment.md [hunk 1]
```diff
@@ -1,57 +1,61 @@
...
 $ py.test tests
 ```
 
-### Check Any Logs
+#### Checking Logs
+
+Logs can be found on the Vagrant guest at the following locations:
 ```
 /opt/postmaster/logs/postmaster.log
 /var/log/apache2/postmaster.access.log
```
Comment (thatarchguy) [inline docs/Development/DevEnvironment.md]: This will be changed with #151

**Comment by PrahlM93** [comment]
@thatarchguy, I just fixed the log location after the latest merge. Take a look at this when you can.

**Pull request merged**